src/gui/MixerView.cpp

```diff
@@ -476,12 +476,20 @@ bool MixerView::confirmRemoval(int index)
 
 void MixerView::deleteUnusedChannels()
 {
-	Mixer* mix = Engine::mixer();
+	TrackContainer::TrackList tracks;
+	
+	auto& songTracks = Engine::getSong()->tracks();
+	tracks.insert(tracks.end(), songTracks.begin(), songTracks.end());
+	
+	auto& patternStoreTracks = Engine::patternStore()->tracks();
+	tracks.insert(tracks.end(), patternStoreTracks.begin(), patternStoreTracks.end());
```

Comment:
Unused variables, maybe artifacts after resolving conflicts?

Reply:
Not sure what happened, but I did a quick rebase with ``master`` to try and solve this.